tutorials/users/ide/version-control.md

```diff
@@ -1 +1,614 @@
-# version-control
+# Version Control
+
+[_Version control_](https://en.wikipedia.org/wiki/Version_control) is a powerful way to organize, back up,
+and share your research outputs and methodologies (code + data) with collaborators, or the public at large.
+A version control system keeps track of a set of files and saves snapshots (i.e. _versions_, _commits_) of 
+the files at any point in time. Using version control allows you to confidently make changes to your code 
+(and any other files), with the ability to roll back to any previous state.
+
+
+## Introduction to Git
+
+Git is one of the most popular version control systems in use today. It has a native command line interface but there 
+are also many third party desktop GUI (Graphical User Interface) clients that help make it easier to get started with Git.  A comprehensive list of 
+several freely available Git clients is [available here](https://git-scm.com/downloads/guis). Online Git services like [GitHub](https://github.com/), [GitLab](https://about.gitlab.com/), or [BitBucket](https://bitbucket.org/) allow you to store your repository online and access it 
+anywhere with a free account.
+
+In this tutorial, you'll learn how to set up a local Git repository, i.e. inside your Marble JupyterLab environment, to version control your work, and how to connect that repository to an online service to backup and share your work (for this tutorial, we will use GitHub as an example).
+
+
+## Setting up Git on Marble
+
+In this step you will configure the local "git" program that will actually do the version control work for you. You will also configure 
+it to connect to your remote repository (optional, but highly encouraged step).
+
+<!-- Now that you have set up a remote repository which you can use to version control your work, backup, and share your
```

Comment:
Can we remove this?

Reply:
That should have been removed in a commit earlier today